api/main.go

```diff
@@ -55,13 +56,22 @@ func main() {
 		log.SetOutput(logger)
 
 		awsConfig := defaults.Get().Config
+		customRetryer := config.NewRetryer(cfg.MinRetryTime(), cfg.MaxRetryTime())
+		awsConfig.Retryer = customRetryer
```

Comment:
From our discussion last week: We actually don’t need a custom retryer. The default exp backoff logic is sufficient; we only need to set the MaxRetries field to something like 999 (so it doesn’t use the “service default” - see sdk docs for more details there).

Reply:
Sorry, I missed this conversation. What's wrong with exposing some of the config for the retryer logic? Part of the benefit of doing so is that we could easily tweak defaults while testing.

Reply:
The backoff retry logic was never an issue, the rate of api calls was. 

In a broader sense, we found the "correct" solution in the 2nd iteration of this problem, but forgot to cleanup the mess left by the first iteration.